### PR TITLE
ENH: Add FIL/UCL OPM data format 

### PR DESCRIPTION
**Pull request by georgeoneill**
In preparation for issue [11257](https://github.com/mne-tools/mne-python/issues/11257)  in MNE-Python.

### CODE REVIEW

File: FIL/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+data_folder
+│
+├───[FILE_PREFIX]_meg.bin             [REQUIRED]
+├───[FILE_PREFIX]_meg.json            [REQUIRED]
+├───[FILE_PREFIX]_channels.tsv        [REQUIRED]
+├───[FILE_PREFIX]_positions.tsv       [OPTIONAL]
+└───[FILE_PREFIX]_coordsystem.json    [OPTIONAL]
+
+```
+However, distinct to BIDS we store no dupliate metadata within our `_meg.bin` file, all metadata is provided within the BIDS-mandated supplementary files.
```
Comment (agramfort) [inline FIL/README.md:26]:
```suggestion
However, distinct to BIDS we store no duplicate metadata within our `_meg.bin` file, all metadata is provided within the BIDS-mandated supplementary files. 
```

File: FIL/sub-noise_ses-001_task-noise220622_run-001_positions.tsv [hunk 2]
```diff
@@ -0,0 +1,69 @@
+name	Px	Py	Pz	Ox	Oy	Oz
```
Comment (agramfort) [inline FIL/sub-noise_ses-001_task-noise220622_run-001_positions.tsv:1]: are these column names part of the standard?

Comment (georgeoneill) [reply FIL/sub-noise_ses-001_task-noise220622_run-001_positions.tsv:1]: No, this is a "manufacturer specific" file. We took a guess on this before BIDS-MEG was finalised. I'll update the README to reflect this.

**Review by agramfort — APPROVED** [review]
@larsoner I let you merge if happy

🙏 @georgeoneill

**Comment by larsoner** [comment]
0.141 released with md5sum
08b1e81e944de9616a408eb70d73775b

**Pull request merged**